bits/sendgrid/sendgrid.go

```diff
@@ -0,0 +1,112 @@
+package sendgrid
+
+import (
+	"context"
+	"net/http"
+
+	"encore.dev"
+	"encore.dev/beta/errs"
+	"encore.dev/pubsub"
+	"encore.dev/rlog"
+	"github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+// Encore docs about secrets: https://encore.dev/docs/primitives/secrets
+var secrets struct {
+	SendGridAPIKey string
+}
+
+type SendParams struct {
+	From struct {
+		Name  string `json:"name,omitempty"`
+		Email string `json:"email"`
+	} `json:"from"`
+	To struct {
+		Name  string `json:"name,omitempty"`
+		Email string `json:"email"`
+	} `json:"to"`
```

Comment:
These still need to be broken out into a separate `Address` type

Reply:
Changed 
b76a820c30d390aea1a3f04fca74168f263ed0a7